pkg/transparentproxy/kubernetes/kubernetes_test.go

```diff
@@ -148,6 +149,9 @@ var _ = Describe("kubernetes", func() {
 				"--exclude-outbound-ports", "11000",
 				"--verbose",
 				"--skip-resolv-conf",
+				"--exclude-outbound-tcp-ports-for-uids", "11003:3",
+				"--exclude-outbound-udp-ports-for-uids", "11001:1",
+				"--exclude-outbound-udp-ports-for-uids", "11002:2",
```

Comment:
Defining multiple times the same param will work?

Reply:
yes, it's an ArrayString, it allows us to define multiple unrelated ranges